Makefile

```diff
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.7
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.8
```

Comment:
```suggestion
DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.10
```

Reply:
I have used 1.10 already in the latest commit